libs/wire-subsystems/src/Wire/UserSubsystem/Interpreter.hs

```diff
@@ -113,6 +140,14 @@ interpretUserSubsystem = interpret \case
   IsBlocked email -> isBlockedImpl email
   BlockListDelete email -> blockListDeleteImpl email
   BlockListInsert email -> blockListInsertImpl email
+  UpdateTeamSearchVisibilityInbound status ->
+    updateTeamSearchVisibilityInboundImpl status
+  SearchUsers luid query mDomain mMaxResults ->
+    searchUsersImpl luid query mDomain mMaxResults
+  BrowseTeam uid browseTeamFilters mMaxResults mPagingState ->
+    browseTeamImpl uid browseTeamFilters mMaxResults mPagingState
```

Comment:
This isn't a blocker for me.

Reply:
Makes sense actually, but I think this would make a very weird first thing to implement in a `TeamSubsystem`. Should we add a note here and move it later? Same for `UpdateTeamSearchVisibilityInbound`.